ReactNative/ReactTwitter/core/views/login-screen.js

```diff
@@ -11,18 +11,16 @@ var Button = require('react-native-button')
 var TwitterRequestsService = require('../service/twitter-requests-service.js')
 var DeepLinkingFacade = require('../service/deep-linking-facade')
 var OauthHelper = require('../service/oauth-helper.js')
-var AuthenticationService = require('../service/authentication-service.js')
 
 var LoginScreenView = React.createClass({
   propTypes: {
-    navigator: React.PropTypes.instanceOf(Navigator).isRequired
+    navigator: React.PropTypes.instanceOf(Navigator).isRequired,
+    twitterService: React.PropTypes.instanceOf(TwitterRequestsService).isRequired
```

Comment:
It's just a constraint for validation, the instance is sent using the component's `props` from here: https://github.com/novoda/spikes/blob/react/PT-70/tweets-list-oauth-integration/ReactNative/ReactTwitter/core/views/main-navigator.js#L37